09-features.md

```diff
@@ -6,6 +6,8 @@ the `channel_announcement` and `node_announcement` messages ([BOLT
 #7](07-routing-gossip.md)).  The flags are tracked separately, since
 new flags will likely be added over time.
 
+Some features were introduced and became so widespread they are `ASSUMED` to be present by all nodes, and can be safely ignored (and the semantics are only defined in prior revisions of this spec).
```

Comment:
Wouldn't we better be served by just starting to set the feature bits to required? That alone might trigger some unexpected behavior in the network that shows maybe we have more stragglers than we thought. 

As an example, we _assumed_ that `max_htlc` was the new default, but then ended up inadvertently partitioning the network as (due to a desynchronized deployment across impls) lnd would still forward `channel_update` messages without that field, which caused CLN to hang up on it. In this instance, couldn't we have start to set the `max_htlc` field in a _new_ even TLV type, which then would have had the opposite effect: old nodes failing (in theory) to parse the new messages.

Reply:
Agreed, this is a first step that implementation should probably take right now.